smoke-tests-otel-starter/spring-boot-3/src/test/java/io/opentelemetry/spring/smoketest/GraalVmNativeMongodbSpringStarterSmokeTest.java

```diff
@@ -24,6 +24,6 @@
     classes = {OtelSpringStarterSmokeTestApplication.class, SpringSmokeOtelConfiguration.class},
     webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @EnabledInNativeImage // see JvmMongodbSpringStarterSmokeTest for the JVM test
-@EnabledInGithubActions
+@RequiresDockerComposeEnvVariable
```

Comment:
just a thought on the name, `@RequiresDockerCompose` would be much clearer to me (and if I wanted to understand the specifics of the condition I could drill into the class)